charts/dapr/charts/dapr_scheduler/templates/_helpers.tpl

```diff
@@ -63,3 +63,25 @@ Create etcd client ports list dynamically based on replicaCount.
 {{- $etcdClientPorts -}}
 {{- end -}}
 
+{{/*
+Create etcd client http ports list dynamically based on replicaCount.
+*/}}
+{{- define "dapr_scheduler.etcdclienthttpports" -}}
+{{- $etcdClientHttpPorts := "" -}}
+{{- $namespace := .Release.Namespace -}}
+{{- $replicaCount := int .Values.replicaCount -}}
+{{- range $i, $e := until $replicaCount -}}
+{{- $instanceName := printf "dapr-scheduler-server-%d" $i -}}
+{{/*{{- $svcName := printf "%s.%s" $instanceName $namespace -}}*/}}
+{{- $clientPort := int $.Values.ports.etcdHttpClientPort -}}
```

Comment:
This looks good since its just changing the port var from the logic above I've thoroughly tested. I'm not sure if there is a way to pass in a var in the helm template helper logic.